spec/clearance/session_spec.rb

```diff
@@ -30,6 +30,17 @@
     expect(session.current_user).to be_nil
   end
 
+  it 'sets a custom cookie name' do
+    Clearance.configuration.cookie_domain = 'custom_token'
```

Comment:
I noticed this project uses primarily single quotes, are you slowly moving this over to double quotes or should I leave this?